postgraphile/src/config/parse.ts

```diff
@@ -19,16 +21,18 @@ export function parseConfig(
   let database = '';
   let user = '';
   let password = '';
+  let schemas = ['public'];
 
   if (configPath) {
-      const tomlContents = readCallback(`${configPath}`).toString();
-      const parsedToml = tomlParseCallback(tomlContents);
-
-      host = parsedToml['database']['hostname'];
-      port = parsedToml['database']['port'];
-      database = parsedToml['database']['name'];
-      user = parsedToml['database']['user'];
-      password = parsedToml['database']['password'];
+    const tomlContents = readCallback(`${configPath}`).toString();
+    const parsedToml = tomlParseCallback(tomlContents);
+
+    host = parsedToml['database']['hostname'];
+    port = parsedToml['database']['port'];
+    database = parsedToml['database']['name'];
+    user = parsedToml['database']['user'];
+    password = parsedToml['database']['password'];
+    schemas = parsedToml['database']['schemas'];
```

Comment:
do we need to update or modify the code below that's assigning based on `process.env`?

Reply:
Would think we don't *need* to, since the schemas var defaults to `["public"]`. We could at some point, but my understanding is that an array env var will introduce further complications

Reply:
Will merge for now, but happy to revisit :)